netket/optimizer/solver/solvers.py

```diff
@@ -18,6 +18,54 @@
 from netket.jax import tree_ravel
 
 
+def eigh(A, b, rcond=1e-14, rcond_smooth=1e-14, x0=None):
```

Comment:
can we rename it to pinv ? eigh is a bit ambigous I find! maybe we should also cite a proper reference for this solver? I saw it used in https://arxiv.org/abs/2212.11289 but maybe it was used before that too?

Reply:
The reason I did not want to call it `pinv` is because this is not `jnp.linalg.pinv` (which I think uses LU factorisation) but a _custom implementation_ of pinv based on the eigendecomposition... 

Though I agree that morally it is a pseudo inverse...

Reply:
ok, then maybe smooth_pinv or similar? it's just that eigh is really the name used for eigendecomposition so it's not very specific